0-bootstrap/main.tf

```diff
@@ -272,3 +277,16 @@ resource "google_folder_iam_member" "folder_tf_compute_security_resource_admin"
 #   role   = "roles/browser"
 #   member = "serviceAccount:${module.jenkins_bootstrap.jenkins_agent_sa_email}"
 # }
+
+
+#Groups creation resources
+data "google_organization" "org" {
+  organization = var.org_id
+}
+
+module "google_groups" {
+  count                = var.create_groups_holder.create_groups ? 1 : 0
+  source               = "./modules/groups"
```

Comment:
Since the module is small, let's move this to a groups.tf instead of wrapping it in a module.